pulp_container/app/viewsets.py

```diff
@@ -545,6 +545,57 @@ class ContainerRepositoryVersionViewSet(RepositoryVersionViewSet):
     """
 
     parent_viewset = ContainerRepositoryViewSet
+    permission_classes = (access_policy.RepositoryVersionAccessPolicyFromDB,)
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_repo_attr_model_or_obj_perms:container.view_containerrepository",
+            },
+            {
+                "action": ["destroy"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_delete_containerrepository_versions_perms",
+            },
+            # {
+            #     "action": ["destroy"],
+            #     "principal": "authenticated",
+            #     "effect": "allow",
+            #     "condition":
+            #         "has_repo_attr_model_or_obj_perms:container.delete_containerrepository",
+            # },
+            # {
+            #     "action": ["destroy"],
+            #     "principal": "authenticated",
+            #     "effect": "allow",
+            #     "condition":
+            #         "has_repo_attr_model_or_obj_perms:container.sync_containerrepository",
+            # },
+            # {
+            #     "action": ["destroy"],
+            #     "principal": "authenticated",
+            #     "effect": "allow",
+            #     "condition":
+            #         "has_repo_attr_model_or_obj_perms:container.build_image_containerrepository",
+            # },
+            # {
+            #     "action": ["destroy"],
+            #     "principal": "authenticated",
+            #     "effect": "allow",
+            #     "condition":
+            #         "has_repo_attr_model_or_obj_perms:container.modify_content_containerrepository",
+            # },
```

Comment:
This is option two. 
A default policy can't be changed easily and there are many statements for the same action/principal/effect.
A good thing is that a user will see those perms in detail if they try to read the access policy via API.

Reply:
Option three is to put all conditions in one loooong string ad have one item for a  `destroy` action.

Reply:
Option 4:
```json
            {
                "action": ["destroy"],
                "principal": "authenticated",
                "effect": "allow",
                "condition":
                    "has_repo_attr_model_or_obj_perms:container.delete_containerrepository"
                    " or has_repo_attr_model_or_obj_perms:container.delete_version_containerrepository",
            },
```